cmd/create/tuningconfigs/spec.json

```diff
@@ -0,0 +1,14 @@
+{
+    "profile": [
+      {
+        "data": "[main]\nsummary=Custom OpenShift profile\ninclude=openshift-node\n\n[sysctl]\nvm.dirty_ratio=\"65\"\n",
+        "name": "tuned-2-profile"
+      }
+    ],
+    "recommend": [
+      {
+        "priority": 20,
+        "profile": "tuned-2-profile"
+      }
+    ]
+ }
```

Comment:
NP: lets add new lines at the end of this file, and the `spec.yaml` file

Reply:
Thanks @hunterkepley , added new lines